.ci/docker/buildtools-windows/README.md

```diff
@@ -18,6 +18,18 @@ docker run --rm -ti \
        msbuild
 ```
 
+## Debugging
```

Comment:
Not related to this particular section but for the previous one regarding the `How to use it`, is https://github.com/elastic/apm-agent-dotnet/pull/689/files#diff-a647f3cfc357b5004f4b869a4b864613R10 still a valid approach? Or does it require to be updated with the new steps to build/run the new dockerfile?